api/src/public/login/script.js

```diff
@@ -47,6 +47,26 @@ const submit = function(e) {
   });
 };
 
+const requestSSOLogin = function(e){
```

Comment:
Aligned this with the logic for the normal logic button from the `submit` function above.

Now it handles denouncing, showing the loading indicator, and displaying errors from the `/medic/login/oidc/authorize` call.  (Note that this does not handle errors that might happen when the OIDC provider redirects the browser back to `/medic/login/oidc` with the auth code.  At that point we have left the login page. Errors from that endpoint will redirect back to the login page with the `sso_error` query param and will be handled by the logic below).

Reply:
This is clean @jkuester. Only redirect to token endpoint in oidc provider from the web app. :clap: